agimus_controller_ros/controller_base.py

```diff
@@ -64,15 +81,21 @@ def __init__(self) -> None:
         self.x_guess = np.zeros(self.nq + self.nv)
         self.u_guess = np.zeros(self.nv)
         self.state_subscriber = rospy.Subscriber(
-            "robot_sensors",
+            self.params.ros_namespace + "/robot_sensors",
```

Comment:
Here you shouldn't do that. The previous version should already take care of prepending the topic name by the ros_namespace.